app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  validates :firstname, presence: true
```

Comment:
We can write it in 1 line. Also, we can add new line after devise to make code mode readable. 
```suggestion

  validates :firstname, :lastname, :email, :password, presence: true
```